metalsmith.json

```diff
@@ -33,14 +33,6 @@
       "source": "./assets",
       "dest": "."
     },
-    "metalsmith-sass": {
-      "outputStyle": "expanded",
-      "includePaths": [
-        "assets/vendor",
-        "assets/vendor/bootstrap-sass-twbs/assets/stylesheets",
-        "assets/vendor/fontawesome/scss/"
```

Comment:
Are we doing this because the font references are different? While Grunt-sass is good, I think there's some things we could get working in metalsmith-sass.